src/pyOpenMS/pxds/ConsensusMap.pxd

```diff
@@ -72,59 +72,59 @@ cdef extern from "<OpenMS/KERNEL/ConsensusMap.h>" namespace "OpenMS":
         ConsensusFeature & operator[](int) nogil except + #wrap-upper-limit:size()
         void push_back(ConsensusFeature spec) nogil except +
 
-        ConsensusMap appendRows(ConsensusMap) nogil except +
-        ConsensusMap appendColumns(ConsensusMap) nogil except +
+        ConsensusMap appendRows(ConsensusMap) nogil except + # wrap-doc:Add consensus map entries as new rows.
+        ConsensusMap appendColumns(ConsensusMap) nogil except + # wrap-doc:Add consensus map entries as new columns.
 
-        void clear(bool clear_meta_data) nogil except +
+        void clear(bool clear_meta_data) nogil except + # wrap-doc:Clears all data and meta data
         void clear() nogil except +
 
-        void updateRanges() nogil except +
+        void updateRanges() nogil except + # wrap-doc:Docu in base class
```

Comment:
That probably does not make much sense in python

Reply:
Please remove